internal/server/iigointernal/judiciary.go

```diff
@@ -15,6 +15,7 @@ import (
 // to be moved to paramters
 const sanctionCacheDepth = 3
 const historyCacheDepth = 3
+const maxNoReport = 500
 
 // to be changed
 const sanctionLength = 2
```

Comment:
when?

Reply:
Yup, raising an issue and will address in next PR (Issue #262 )